docs/_sass/minima/_base.scss

```diff
@@ -288,7 +288,7 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "TutorAid";
```

Comment:
Wonder what this change does? A bit concerned with merging this because I don't know what it's for

Reply:
I believe this is part of updating the site-wide settings for our project. I am not entirely sure of their effects either but I was following the guide on our course site for the 'README' section and they say that this comes into play when you convert documentation to PDF formats (shown in the image below).
<img width="200" alt="Instructions" src="https://user-images.githubusercontent.com/65402015/134512564-c0f34ed1-f8a1-4826-9f56-275b4e70180e.png">

Reply:
Thanks for clarifying!